vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/FieldSteps.java

```diff
@@ -88,6 +95,29 @@ public WebElement clearFieldLocatedByUsingKeyboard(SearchAttributes locator)
         return element;
     }
 
+    /**
+     * Gets a list of elements by <b>locator</b> and check that it is
+     * sorted by text with using <b>sortingOrder</b> by element text.
+     * List should have more than 1 elements.
+     * @param locator to locate elements
+     * @param sortingOrder The set of sort's rule<br>
+     * <i>Available rule:</i>
+     * <ul>
+     * <li><b>ASCENDING</b> - will check that the list is sorted alphabetically,
+     * <li><b>DESCENDING</b> - will check that the list is sorted reverse alphabetically
+     * </ul>
+     */
+    @Then("elements located `$locator` are sorted by text in $sortingOrder order")
+    public void doesElementSorted(SearchAttributes locator, SortingOrder sortingOrder)
+    {
+        List<WebElement> elements = baseValidations.assertIfAtLeastNumberOfElementsExist(
+                String.format("Fields by locator: %s", locator), locator, 2);
+        List<String> listText = elements.stream().map(WebElement::getText).collect(Collectors.toList());
```

Comment:
- WebElementActions.getElementText
- don't use collection names in variable names